src/Argu/paket.template

```diff
@@ -5,21 +5,26 @@ description
 authors
     Eirik Tsarpalis
 copyright
-	Copyright 2017
+    Copyright 2017
 summary
     A declarative command line and XML configuration parser for F# applications.
 licenseurl https://github.com/fsprojects/Argu/blob/master/License.md
 projecturl http://fsprojects.github.io/Argu/
 iconurl https://raw.githubusercontent.com/fsprojects/Argu/master/docs/files/img/logo.png
 include-pdbs true
 tags
-    F#, argument, parser
+    F#, argument, parser, cli, commandline
 references
     Argu.dll
 files
-    ../../bin/net40/Argu.dll ==> lib/net40
-    ../../bin/net40/Argu.pdb ==> lib/net40
-    ../../bin/net40/Argu.xml ==> lib/net40
-    ../../bin/net35/Argu.dll ==> lib/net35
-    ../../bin/net35/Argu.pdb ==> lib/net35
-    ../../bin/net35/Argu.xml ==> lib/net35
+    ./../../bin/net461/Argu.dll ==> lib/net461
+    ./../../bin/net461/Argu.pdb ==> lib/net461
+    ./../../bin/net461/Argu.xml ==> lib/net461
+    ./../../bin/net40/Argu.dll  ==> lib/net40
+    ./../../bin/net40/Argu.pdb  ==> lib/net40
+    ./../../bin/net40/Argu.xml  ==> lib/net40
+    ./../../bin/net35/Argu.dll  ==> lib/net35
+    ./../../bin/net35/Argu.pdb  ==> lib/net35
+    ./../../bin/net35/Argu.xml  ==> lib/net35
+excludeddependencies
+    FSharp.NET.Sdk
```

Comment:
@enricosada I don't know why it's in there, you can  see I put it in the excluded dependencies